docs/schema.md

```diff
@@ -155,19 +155,19 @@ string of the format `<DB>-<ENTRYID>`, where `DB` names the database and
 
 The defined database prefixes and their "home" databases are:
 
-| Prefix | Description |
-| ------ | ----------- |
-| `A` | The [Android vulnerability database](TODO). Serving the shared format [here](https://storage.googleapis.com/android-osv/). |
-| `GO` | The [Go vulnerability database](https://github.com/golang/vulndb). Serving the shared format [here](https://storage.googleapis.com/go-vulndb/). |
-| `OSV` | The <https://osv.dev> vulnerability database. Serving `<ID>` in the shared format at `https://api.osv.dev/v1/vulns/<ID>` |
-| `PYSEC` | The [PyPI vulnerability database](https://github.com/pypa/advisory-db). Serving `<ID>` in the shared format at `https://api.osv.dev/v1/vulns/<ID>` |
-| `RUSTSEC` | [The Rust crates vulnerability database](https://github.com/rustsec/advisory-db). Serving `<ID>` in the shared format at  `https://github.com/RustSec/advisory-db/blob/osv/crates/<ID>.json` |
-| `GSD` | The GSD database. Serving the shared format [here](https://github.com/cloudsecurityalliance/gsd-database). |
-| `GHSA` | The GitHub Security Advisory database. Serving the shared format [here](https://github.com/github/advisory-database). |
-| `LBSEC` | The LoopBack Advisory Database. Serving the shared format [here](https://github.com/loopbackio/security/tree/main/advisories). |
-| `DSA`/`DLA`/`DTSA` | The [Debian Security Advisory](https://www.debian.org/security/) database. Serving the shared format [here](https://storage.googleapis.com/debian-osv/dsa-osv/). |
-| `RLSA`/`RXSA` | The [Rocky Linux Security Advisory](https://errata.rockylinux.org) database. Serving the shared format [here](https://apollo.build.resf.org/api/v3/osv/)
-| Your database here | [Send us a PR](https://github.com/ossf/osv-schema/compare). |
+| Prefix | Database | ID Canonical URL | ID API Endpoint | ID Regex |
```

Comment:
Are ID canonical URL and ID regexes useful enough to most users to warrant columns here? 

We did want to extend this, but do this in a way that doesn't introduce too many columns to overwhelm the general reader. One very useful column we wanted to add was a "How to contribute" column that points to how downstream consumers can submit improvements to these (e.g. GitHub, Go, Rust all have their own contribution process).

Reply:
GSD, for example, uses the canonical URL to auto-link to aliases/related IDs for human users. Similarly, the Regex can be used to know when an ID matches a particular database namespace. It's not going to be useful for all consumers, but it does provide enough value IMO to be worth the inclusion.

For example, not all IDs follow the typical `PREFIX-\d{4}-\d{4,}` regex - See: PYSEC, GHSA, DSA, ALSA, RLSA, etc. This could throw off consumers who expect all IDs to use the same format.

I 100% agree on the columns issue, especially with the rendered page being limited to readable width... Also, agree on the contribution link; that would be super valuable.

I'll see if I can figure out markdown tables with multiple lines... The simplest solution would be to just use two columns and embed this information in the description column rather than having 5+ columns.

Reply:
> For example, not all IDs follow the typical PREFIX-\d{4}-\d{4,} regex - See: PYSEC, GHSA, DSA, ALSA, RLSA, etc. This could throw off consumers who expect all IDs to use the same format.

What would be the use case for a consumer to want to apply a regex to the IDs they get? The bits that come after just needs to be treated as a unique identifier. Why would a consumer want to validate beyond this prefix, other than treat them as opaque identifiers?

Reply:
If you're extracting information from the ID (e.g., year), knowing the ID is in the correct format is useful to avoid parsing errors. It's also helpful to see the format of an ID in general, although that can also be accomplished with an example ID instead of the regex. Regex does two birds with one stone, however.

If it's a blocker, I can remove the regexes and track those in the GSD docs instead.